isaric/parsers/isaric-core.toml

```diff
@@ -550,6 +550,21 @@
     { field = "flw2_date_death", description = "If known, date of death" },
   ]
 
+[subject.vaccinated_covid19]
+  field = "vacc_cov19_cmyn"
+  can_skip = true
+  values = { 1 = true, 0 = false }
+
+[subject.vaccinated_covid19_dates]
+  combinedType = "set"
+  excludeWhen = "none"
```

Comment:
combined type here = set, should we use set (to remove duplicates) for all parsers (previous parsers here indicate combined type = list) - we can probably assume that any duplicated vaccination dates are incorrect, although the presence of multiple dates can indicate the number of doses received even if the dates are unreliable. I think I'd suggest keeping all dates so use combined type = list

Reply:
discussed in meeting, keep as 'set' where indicated as set - Pip has identified duplicates in these specific parsers, schema set up is to remove duplicates and should not be changed for this